dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1,6 +1,7 @@
 package com.dotcms.content.elasticsearch.business;
 
 import com.dotcms.business.CloseDBIfOpened;
+import com.dotmarketing.business.IdentifierCache;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.IdentifierCache'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)